config/locales/client.en.yml

```diff
@@ -5468,6 +5468,7 @@ en:
             Specify the city that will be used as the jurisdiction for resolving any disputes related to the use of this forum. This information is typically included in legal documents such as the forum's Terms of Service.
           optional: "(optional)"
           update: "Update"
+          saved: "saved!"
```

Comment:
We can remove this if we change to toasts

Reply:
I need to provide a message for the toast, what should the toast say?

Reply:
Can we do it based on the section? Eg. "General settings saved", "Contact information saved" etc.? If that's too annoying just do "About configuration saved"